content/en/blog/_posts/2020-08-25-gsoc‘20 -building-operators-for-cluster-addons.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: blog
+title: "GSoC 2020 - Building operators for cluster addons"
+date: 2020-08-25
+slug: gsoc20-building-operators-for-cluster-addons
+---
+
+**Author**: Onyekwere Somtochi
+
+# Introduction
+
+Google Summer of Code is a global program that is geared towards introducing students to open source. Students are matched with open-source organizations to work with them for three months during the summer.
+
+My name is Somtochi Onyekwere from the Federal University of Technology, Owerri(Nigeria) and this year, I was opportune to be selected as one of the interns to work with Kubernetes(under the CNCF organization) and this led to an amazing summer spent learning, contributing and interacting with the community.
```

Comment:
Probably want this to say "I was given the opportunity to work with Kubernetes" instead of "I as opportune..."